build/yamls/flow-visibility/patches/release/.gitignore

```diff
@@ -0,0 +1 @@
+# placeholder
```

Comment:
Is this file needed for this PR? Not a big deal, I just don't fully understand why it's needed in the first place

Reply:
It seems it works without this file, but I just add it to keep consistence with the flow aggregator and antrea patches.